### DISCUSSION TITLE
Can I pass configuration as arguments?

### DISCUSSION THREAD

**Discussion by phdpe** [any1/wayvnc#160] [Q&A]
I have a very simple configuration file that I would like to simply call it on command line.  

For example, can I start something like `wayvnc --port=9999` rather than put `port=9999` in a separate file? 

Thanks!

**Comment by lack** [comment]
In wayvnc 0.5.0 and later you can:

```
wayvnc 127.0.0.1 9999
```

See `wayvnc --help` for more options.

**Reply by any1** [reply]
It's been possible from very early on, if not from the start to set the listening address and port.
